packages/firestore/test/integration/api/database.test.ts

```diff
@@ -669,4 +665,17 @@ apiDescribe('Database', persistence => {
         .then(() => awaitOnlineSnapshot());
     });
   });
+
+  it('can enable and disable networking', () => {
```

Comment:
I don't think I saw this test in the Android version.  Can you either add it there, or remove it here (I don't care strongly which)?

Reply:
Yes, I haven't done that yet. I will address other clients once I get this one in an acceptable state.